SiennaOpenAPIModels.jl/src/dbinterface/sqlite.jl

```diff
@@ -189,8 +203,16 @@ function load_to_db!(db, data)
         DBInterface.execute(
             db,
             "INSERT INTO attributes (entity_id, entity_type, key, value)
-VALUES (?, ?, ?, ?)",
-            [data["id"], "", k, v],
+VALUES (?, ?, ?, jsonb(?))",
```

Comment:
What are we ingesting here? j

Reply:
Added a comment explaining that we are adding the attributes and we need to write out JSON for SQLite dynamic typing.